custom_theme/ganache.html

```diff
@@ -131,52 +123,52 @@ <h2>ONE CLICK BLOCKCHAIN</h2>
                 <br/>
                 <span class="os">(Windows)</span>
                 </p>
-                <img src="/img/ganache/windows-logo.svg" />
+                <img alt="" src="/img/ganache/windows-logo.svg" />
             </a>
-            <p class="ganache-other-os first"><a href="https://github.com/trufflesuite/ganache-ui/releases">Need another OS download?</a></p>
+            <p class="ganache-other-os first"><a target="_blank" rel="noopener noreferrer" href="https://github.com/trufflesuite/ganache-ui/releases">Need another OS download?</a></p>
```

Comment:
it's funny that it is `noreferrer` - I think the actual field is the misspelling, "referer"